torchx/schedulers/ray_scheduler.py

```diff
@@ -397,7 +397,16 @@ def log_iter(
             return iterator
 
         def list(self) -> List[str]:
-            raise NotImplementedError()
+            address = os.getenv("RAY_ADDRESS")
+            if address is None:
+                raise Exception(
+                    "RAY_ADDRESS env variable is expected to be set to list jobs on ray scheduler"
```

Comment:
What about just keeping "RAY_ADDRESS env variable is missing" message?

Reply:
May be, but if we are also going to throw for empty value for RAY_ADDRESS, I'm thinking of a message like "RAY_ADDRESS env variable is expected to be set validly"